docs/dataloaders.rst

```diff
@@ -22,6 +22,12 @@ UserCSVLoader
 .. autoclass:: deepchem.data.UserCSVLoader
   :members:
 
+JsonLoader
+^^^^^^^^^^
+
```

Comment:
Could you add a few sentences here describing when we'd want to use `JsonLoader` as opposed to other types of loaders?

Reply:
Sure, I'll add some description of JSON and when it might be advantageous to use.